libraries/Microsoft.Bot.Builder.LanguageGeneration/Analyzer.cs

```diff
@@ -104,6 +104,26 @@ public override List<string> VisitConditionalBody([NotNull] LGFileParser.Conditi
             return result;
         }
 
+        public override List<string> VisitSwitchCaseBody([NotNull] LGFileParser.SwitchCaseBodyContext context)
+        {
+            var result = new List<string>();
+            var switchCaseNodes = context.switchCaseTemplateBody().switchCaseRule();
+            foreach (var iterNode in switchCaseNodes)
+            {
+                var expression = iterNode.switchCaseStat().EXPRESSION();
```

Comment:
May expression be undefined if there is no expression after case?

Reply:
staticchecker handles verifying whether there is a valid expression after SWITCH and CASE. If not, it will prompt an error.

Reply:
I see.